addons/website/models/website_controller_page.py

```diff
@@ -0,0 +1,64 @@
+
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.http_routing.models.ir_http import slugify
+from odoo import api, fields, models
+
+
+class WebsiteControllerPage(models.Model):
+    _name = 'website.controller.page'
+    _inherits = {'ir.ui.view': 'view_id'}
```

Comment:
From what I could grasp, this was a controversial thing to have on website_page.
We don't have to do an inherits I think, just related fields if we  need them